blog/2022-04-07-policy-cli-and-github-packages/2022-04-07-policy-cli-and-github-packages.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: policy-cli-and-github-packages
+title: The Policy CLI and Github Packages
+authors: [daniel]
+tags: [policy, github]
+---
+
+Today, we are happy to announce that the Policy CLI can interact directly with Github Packages (GHCR)!
+
+GitHub Packages is a platform that allows you to host and maintain packages and containers.  The Policy CLI is a tool built by Aserto that can be used for building, versioning, and publishing your authorization policies. It uses OCI standards to manage artifacts, and the [Open Policy Agent (OPA)](https://github.com/open-policy-agent/opa) to compile and run (For more instructions for downloading and using the Policy CLI head to our [documentation](https://www.openpolicyregistry.io/docs/cli/download/)).
+
+You can now use the Policy CLI to manage your policy images with GHCR as well as the Open Container Registry and the Aserto Container Registry. In this post, we’ll walk through using the Policy CLI with GHCR.
```

Comment:
You can now use the Policy CLI to manage your policy images with GHCR as well as the Open Container Registry. In this post, ...